src/pages/ecosystem/Gallery/GalleryItem.tsx

```diff
@@ -41,8 +41,8 @@ const FaceSide = styled(motion.div)(
   box-shadow: 2px 2px 10px 2px rgba(131, 131, 131, 0.4);
   background-color: rgba(249, 249, 249, 0.3);
   padding: 1rem;
-  border-radius: 2rem;
-  background-color: #fff;
+  border-radius: 0.6rem;
```

Comment:
thanks! 

@yttriumli did you mean `0.6rem` by `a radius of 6`? 

@Holybasil could you turn these into reusable values (or partial/styled/components) that only need to be changed in one spot for all instances to get updated, to make it less probable that we'll miss some manually inserted constant/string styling somewhere, and also to make updating later easier?

Reply:
how about adding a new task for this? I remember you required identical tiles all over the website earlier (includes rollupscan page and so on). therefore, it's better to solve this problem separately from this one.
I need to know which tiles require identical box shadows and border radius (maybe anything else). does @yttriumli or @szgbo have time to go over the whole website and unify the tiles?

Reply:
Ok, good point. Could you kindly create a separate Asana card for this and get design input? And let's proceed with this PR then @Holybasil

Reply:
Maybe @stateofkate could help with the standardisation here too btw

Reply:
sure, and have resolved conflict

Reply:
Currently Figma value: corner radius 6 px.

Reply:
Thanks. We probably want to use rem instead of px throughout the project, right? @yttriumli @Holybasil

Reply:
yes